RESTART-59 Extra search parameter for radius of search

mattKendon opened the pull request:
### Description

Set 5-7 starting a mile and up 1, 2, 3, 5 & 10miles

### Action

Added new Field to the search form for radius. 

Added new `config/map.php` to hold radius options and default radius configuration.

Commit:
added radius to form

Commit:
added radius to form

Commit:
post radius when searching on map

Commit:
post radius when searching on map

Commit:
set radiuses using the config

Commit:
set radiuses using the config

Commit:
update migration for fixometer connection for local testing

A new migration has been created that updates the column names to support the new users table.
This was required because the users table that the Repair Directory has changed since the
move of the Fixometer/Restarters application to Laravel.

Commit:
when running testing php service depend on the database_testing service

Commit:
depend on database_testing for the other services as well

ngm commented:
- Could the search dropdown also indicate that the radius is in miles, e.g. 1 mile, 2 miles, etc.
- Is it possible for this to be part of the text of each option in the dropdown, and for the text for each to come from the configuration in config/map.php
- This would have the benefit of allowing us to have the largest search say 'All of London' if we desired

Commit:
Merge branch 'RESTART-59' of github.com:TheRestartProject/RepairDirectory into RESTART-59

Commit:
move the label for the radius select box into the lang file so that custom lanaguage changes can be made.

Commit:
change the way the form is ordered so that on large screen we can make it shorter

mattKendon commented:
I've now updated the radius select field to output text for the label of each option (the value is still the same as a number). This is achieved using the lang file, so if/when this gets used in other languages it will be easy to translate. 

I've updated the form so that on large screen sizes the input fields are side by side as per your diagram. However, on small laptop screens and below the sidebar area is too thin to do this, so I've had to stay with the full width input fields. 

Regarding the zoom levels, Google maps zoom levels are very imprecise. There are only two zoom steps between the fully zoomed out level for london (11) and the default zoom level (13). Because of this I fear that for most radiuses wouldn't have a different zoom level than the default (I certanly wouldn't zoom further in than the default level (13)).
